Source/WebKit/UIProcess/win/AutomationClientWin.h

```diff
@@ -40,14 +42,21 @@ namespace WebKit {
 class AutomationSessionClient final : public API::AutomationSessionClient {
 public:
     explicit AutomationSessionClient(const String&);
+    ~AutomationSessionClient();
 
     String sessionIdentifier() const override { return m_sessionIdentifier; }
 
     void requestNewPageWithOptions(WebKit::WebAutomationSession&, API::AutomationSessionBrowsingContextOptions, CompletionHandler<void(WebKit::WebPageProxy*)>&&) override;
     void didDisconnectFromRemote(WebKit::WebAutomationSession&) override;
 
+    void retainWebView(Ref<WebView>&&);
+    void releaseWebView(HWND);
+
 private:
     String m_sessionIdentifier;
+
+    static void close(WKPageRef, const void*);
+    HashMap<HWND, RefPtr<WebView>> m_webViews;
```

Comment:
We no longer have the reason to use HWND here. We can use WebPageProxy* or NativeWindowHandle.

Reply:
I think we don't need HashMap anymore. We can do `Vector<Ref<WebView>> m_webViews` here.

Reply:
You can use `WebView::page()` to get a `WebPageProxy*` for matching.

Reply:
Thank you for pointing this out.
The `HWND` has been removed as it is no longer needed.
WebViews management has been changed to `HashSet`.